yellow-paper/docs/public-vm/gen/_InstructionSet.mdx

```diff
@@ -30,193 +30,201 @@ Click on an instruction name to jump to its section.
 	}</CodeBlock></td>
 </tr>
 <tr>
-	<td style={{'text-align': 'center'}}>0x02</td>	<td style={{'text-align': 'center'}}><a id='isa-table-div'/><Markdown>[`DIV`](#isa-section-div)</Markdown></td>
+	<td style={{'text-align': 'center'}}>0x02</td>	<td style={{'text-align': 'center'}}><a id='isa-table-mul'/><Markdown>[`MUL`](#isa-section-mul)</Markdown></td>
+	<td><Markdown>Subtraction (a - b)</Markdown></td>
+	<td><Markdown>128</Markdown></td>
+	<td><CodeBlock language="jsx">{
+		`M[dstOffset] = M[aOffset] * M[bOffset] mod 2^k`
+	}</CodeBlock></td>
+</tr>
+<tr>
+	<td style={{'text-align': 'center'}}>0x03</td>	<td style={{'text-align': 'center'}}><a id='isa-table-div'/><Markdown>[`DIV`](#isa-section-div)</Markdown></td>
 	<td><Markdown>Unsigned division (a / b)</Markdown></td>
 	<td><Markdown>128</Markdown></td>
 	<td><CodeBlock language="jsx">{
 		`M[dstOffset] = M[aOffset] / M[bOffset]`
 	}</CodeBlock></td>
 </tr>
 <tr>
-	<td style={{'text-align': 'center'}}>0x03</td>	<td style={{'text-align': 'center'}}><a id='isa-table-eq'/><Markdown>[`EQ`](#isa-section-eq)</Markdown></td>
+	<td style={{'text-align': 'center'}}>0x04</td>	<td style={{'text-align': 'center'}}><a id='isa-table-eq'/><Markdown>[`EQ`](#isa-section-eq)</Markdown></td>
```

Comment:
Thanks for updating this! Hope you had no issues running the script! I haven't push the python scripts I use for image-gen, so I can batch update those again once the instruction set settles.